burgle.lic

```diff
@@ -477,7 +477,7 @@ class Burgle
 
   #ripped out of steal.lic
```

Comment:
Does the same change need to be made to `steal.lic`?

Reply:
Good plan, done, https://github.com/rpherbig/dr-scripts/pull/4959/commits/0ebf637dfe2d2917248ab874ed8be7ee927b8503